### ISSUE TITLE
Support for ATS Heavy Haul

### ISSUE THREAD

**Issue by Simeta** [davidsantos-br/ETS2Sync-Helper-4#12]
Now TMP have support for this update could you update this great app so we can heavy haul in convoys. Thanks.

**Comment by DaviMedrade** [comment]
Sure. Once I get the DLC (which should be in a few days) I should be able to do it pretty quickly.

**Comment by Simeta** [comment]
Any progress? :)

**Comment by DaviMedrade** [comment]
I'm trying to buy the DLC since yesterday but my bank won't cooperate. Their security software won't work on my computer (I've tried to install it about half a dozen times) and I can't pay for the purchase without it. Maybe I'll be able to do it today from another computer.

**Edit**: Finally, it worked. I had to use TeamViewer to do it from my friend's PC. An update should be available in a few hours.
